CHARTER.md

```diff
@@ -0,0 +1,16 @@
+# Common Operating System Interface Charter
+
+The goal of this document is to clearly define what this project claims responsibility over.
+
+### COSI Specification
+
+The project will maintain a community driven API that will represent the boundaries between a distributed orchestration system (such as Kubernetes) and the underlying userland and kernel (such as Linux).
+
+### COSI Implementations
+
+The project will selectively host and mantain various tools that implement the COSI spec. These tools will be independently maintained, and each will declare it's own scope of work and release cycle.
+
+### COSI Open Source
+
+Having a safe, effective, open, transparent, and commercial free experience is our number 1 value. As COSI evolves as a project, we may consider some sort of formal project governance. For now we trust commercially motivated contributions to maintain a clear and transparent set of ethics. 
```

Comment:
super duper nit: `s/number 1/number one`. I think an English teacher got mad at me for this once, but I couldn't tell you why 😄 .